docs/integrations/r.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: default
+title: Using R with lakeFS
+description: How to use lakeFS from R including creating branches, committing changes, and merging.
+parent: Integrations
+nav_order: 60
+has_children: false
+---
+
+# Using R with lakeFS
+
+R is a powerful language used widely in data science. lakeFS interfaces with R in two ways: 
+
+*  To **read and write data in lakeFS** use standard S3 tools such as the `aws.s3` library. lakeFS has a [S3 gateway](https://docs.lakefs.io/understand/architecture.html#s3-gateway) which presents a lakeFS repository as an S3 bucket.
+* For working with **lakeFS operations such as branches and commits** use the [API](https://docs.lakefs.io/reference/api.html) for which can be accessed from R using the `httr` library. 
+
+_To see examples of R in action with lakeFS please visit the [lakeFS-samples](https://github.com/treeverse/lakeFS-samples/) repository and the [sample](https://github.com/treeverse/lakeFS-samples/blob/main/00_notebooks/R.ipynb) [notebooks](https://github.com/treeverse/lakeFS-samples/blob/main/00_notebooks/R-weather.ipynb)_.
+
+{% include toc.html %}
+
+## Reading and Writing from lakeFS with R
+
+Working with data stored in lakeFS from R is the same as you would with an S3 bucket, via the [S3 Gateway that lakeFS provides](https://docs.lakefs.io/understand/architecture.html#s3-gateway).
+
+You can use any library that interfaces with S3. In this example we'll use the [aws.s3](https://github.com/cloudyr/aws.s3/tree/master) library.
+
+```r
+install.packages(c("aws.s3"))
+```
+
+### Configuration 
+
+The [R S3 client documentation](https://github.com/cloudyr/aws.s3/blob/master/man/s3HTTP.Rd) includes full details of the configuration options available. A good approach for using it with lakeFS set the endpoint and authentication details as environment variables: 
```

Comment:
The doc link is to man format. Consider linking the site's referenced documentation link: https://cloud.r-project.org/web/packages/aws.s3/aws.s3.pdf

Reply:
Fair point. I still find it a bit gross that R doesn't publish proper online docs, hence linking to the man format. But I guess PDF is more readable.